src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -181,7 +184,7 @@ class CombatLogParser extends MainCombatLogParser {
         icon={<SpellIcon id={SPELLS.UPLIFTING_TRANCE_BUFF.id} />}
         value={`${formatPercentage(unusedUTProcs)} %`}
         label={(
-          <dfn data-tip={`You got total <b>${this.modules.upliftingTrance.UTProcsTotal} uplifting trance procs</b> and <b>used ${this.modules.upliftingTrance.consumedUTProc}</b> of them. <b>${this.modules.upliftingTrance.nonUTVivify} of your vivify's were used without an uplifting trance proc</b>.`}>
+          <dfn data-tip={`You got total <b>${this.modules.upliftingTrance.UTProcsTotal} uplifting trance Proccs</b> and <b>used ${this.modules.upliftingTrance.consumedUTProc}</b> of them. ${this.modules.upliftingTrance.nonUTVivify} of your vivify's were used without an uplifting trance Proccs or without a Thunder Focus Tea buff.`}>
```

Comment:
Please call them `procs` with lower case and one c. I reckon that's the most commonly used term and would make it consistent with the rest of the app.